fly/commands/login.go

```diff
@@ -172,7 +154,12 @@ func (command *LoginCommand) passwordGrant(client concourse.Client, username, pa
 		return "", "", err
 	}
 
-	return token.TokenType, token.AccessToken, nil
+	idToken, ok := token.Extra("id_token").(string)
+	if !ok {
+		return "", "", errors.New("invalid id_token")
```

Comment:
is it also possible that id_token might be missing so this could be `invalid or missing id_token`